uwrt_mars_rover_arm/uwrt_mars_rover_arm_hw/CMakeLists.txt

```diff
@@ -41,21 +50,26 @@ include_directories(include)
 #	include
 #)
 
+
 # add dynamic library & it's dependecies
 add_library(
 	${PROJECT_NAME}
 	SHARED
 	src/${PROJECT_NAME}_actuator.cpp
     src/${PROJECT_NAME}_differential.cpp
 )
+
 ament_target_dependencies(
 	${PROJECT_NAME}
 	hardware_interface
 	transmission_interface
 	pluginlib
 	rclcpp_lifecycle
 	rclcpp
+  uwrt_mars_rover_utils
```

Comment:
be mindful of formatting. also https://github.com/uwrobotics/uwrt_mars_rover/pull/184#discussion_r813542920 is not resolved still.